tests/fingerprints/functional_groups.py

```diff
@@ -8,7 +8,7 @@
 
 
 def test_functional_groups_bit_fingerprint(smiles_list, mols_list):
-    fg_fp = FunctionalGroupsFingerprint(n_jobs=-1)
+    fg_fp = FunctionalGroupsFingerprint(n_jobs=-1, verbose=0)
```

Comment:
Leave existing tests as they were. Instead, add more tests in `tests/utils/parallel.py`. You can also compute any fingerprints there, as needed. We try to isolate what we test - those are not truly unit tests, but here they should reasonably isolate testing the given fingerprint.

Reply:
Restored as it was earlier